docs/optimizations/FileLocalization.md

```diff
@@ -0,0 +1,52 @@
+# The 'localization_optional' Optimization
```

Comment:
Sounds like the title of a Robert Ludlum novel 😄

Reply:
Or the title of a [Big Bang Theory episode](https://en.wikipedia.org/wiki/List_of_The_Big_Bang_Theory_episodes)